development/components/grid/columns-reference/toggle.md

```diff
@@ -9,6 +9,9 @@ weight: 10
 
 This ToggleColumn is used to display booleans, it will display an icon instead of the value. If the user clicks on it, this triggers a toggle of the boolean value.
 
+**Warning:** **Notice** that the ajax call trigger is subordinated to the loading of the relative js extension (**window.prestashop.component.GridExtensions.ColumnTogglingExtension()**). That is not automatic but have to be done manually if the grid does not have any other Toggle Column by default. In prestashop >1.7.8 you can use the **global component**) to load the extension.
+https://devdocs.prestashop.com/1.7/development/components/global-components/#global-javascript-components
```

Comment:
Use link instead :)

Reply:
Sorry for my poor english ;),what I wanted to do was to warn the documentation user that they need to load the javascript extension to use the Toggle Column component.